doc/docs.md

```diff
@@ -3346,7 +3346,7 @@ and it will always return the same data.
 
 #### $env
 
-```v
+```v ignore
```

Comment:
that should compile, no point of `ignore` imho

Reply:
I am working on fixing vfmt.

Reply:
Ok, I got an error from it - so was unsure.

Reply:
Ah it's vfmt